posthog/models/integration.py

```diff
@@ -172,8 +230,54 @@ def integration_from_oauth_response(
             },
         )
 
+        if integration.errors:
+            integration.errors = ""
+            integration.save()
+
         return integration
 
+    def access_token_expired(self, time_threshold=timedelta(seconds=180)) -> bool:
```

Comment:
Is 180 seconds enough? This seems reasonable if celery truly runs every 60 sec on the spot, but if it's 2min late one day due to some deploys, AWS redis network issues, etc, might this be cutting it close? For a token with an 1h expiry, I'd start reloading at 50min or something more like `0.75 * expires_in` just to be on the safe side.

Reply:
Good point...

Reply:
Updated it. Went for 0.5 to be extra safe